src/vs/workbench/parts/tasks/test/common/problemMatcher.test.ts

```diff
@@ -173,8 +173,6 @@ suite('ProblemPatternParser', () => {
 			];
 			let parsed = parser.parse(problemPattern);
 			assert.equal(null, parsed);
```

Comment:
Shouldn't we not still get the error message. The test ensures that the error message shows
up in the output panel.

Reply:
That error message was coming from `validateProblemPattern` which no longer needs to validate that the regexp exists because we validate it much earlier in parse now. I can add the message in to parse.